test/e2e/istio_csr_test.go

```diff
@@ -99,6 +101,27 @@ var _ = Describe("Istio-CSR", Ordered, Label("TechPreview", "Feature:IstioCSR"),
 		Expect(err).NotTo(HaveOccurred())
 		ns = namespace
 
+		By("creating cluster issuer")
+		loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "self_signed", "cluster_issuer.yaml"), ns.Name)
+		DeferCleanup(func() {
+			loader.DeleteFromFile(testassets.ReadFile, filepath.Join("testdata", "self_signed", "cluster_issuer.yaml"), ns.Name)
+		})
+
+		By("issuing TLS certificate")
+		loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "self_signed", "certificate.yaml"), ns.Name)
+		DeferCleanup(func() {
+			loader.DeleteFromFile(testassets.ReadFile, filepath.Join("testdata", "self_signed", "certificate.yaml"), ns.Name)
+		})
+
+		err = waitForCertificateReadiness(ctx, "my-selfsigned-ca", ns.Name)
+		Expect(err).NotTo(HaveOccurred())
+
+		By("creating istio-ca issuer")
+		loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "istio", "istio_ca_issuer.yaml"), ns.Name)
+		DeferCleanup(func() {
+			loader.DeleteFromFile(testassets.ReadFile, filepath.Join("testdata", "istio", "istio_ca_issuer.yaml"), ns.Name)
+		})
+
```

Comment:
nit: Should these and the ConfigMap creation with proto content be moved to BeforeAll?

Reply:
Ideally should be the case, updating the existing CR for different scenarios, instead of creating anew for every test. This can be worked on in a follow up PR.